src/components/form/fields/DescriptionField.tsx

```diff
@@ -11,17 +49,27 @@ export function DescriptionField({
     helpText?: string
     schemaSection: SchemaSection
 }) {
-    const validate = useCheckMaxLengthFromSchema(schemaSection.name, 'formName')
+    const validator = useValidator({ schemaSection, fieldName: 'description' })
```

Comment:
Should only need useCheckMaxLengthFromSchema .
Description is not required nor unique.

Reply:
I think only the field name was wrong in the old code.